tests/integration/helpers/network.py

```diff
@@ -212,8 +219,8 @@ def _iptables_cmd_suffix(
 
     def __init__(
         self,
-        container_expire_timeout=50,
-        container_exit_timeout=60,
+        container_expire_timeout=120,
```

Comment:
https://s3.amazonaws.com/clickhouse-test-reports/0/0f74d6d14dd01129979dc2361e080da6b150857c/integration_tests__release__[1_4].html

Container was erased in the middle of test